### DISCUSSION TITLE
intensity normalisation & reports

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#347] [Q&A] [answered]
Unfortunately, some aspects regarding the normalization / the reports are not completely clear to me. 

The main report contains 3 types of intensities
- unnormalized
- 'normalized'
- MaxLFQ-normalized

Q1: I presume 'normalized' refers to the basic cross-run normalisation algorithm set in DIA-NN (RT-dependent, global, RT & signal). Correct? Unfortunately that is not explicit in the section on the main report
Q2: Does the MaxLFQ setting mean that the MaxLFQ algorithm is applied to either (i) the unnormalized intensities or (ii) the intensities normalized by the basic cross-run normalisation algorithm ? 
Q3: I presume that (Q2) does make a difference, because MaxLFQ doesn't completely incorporate the logic of the basic cross-run normalization algorithms. Correct? Presumably, MaxLFQ on basic cross-run normalized is better, correct?
Q4: What does the 'Global' type cross-run normalization do? When is it recommended to use this one compared to the RT-dependent one? What are the advantages/disadvantages?

It appears to me that the github description section on the type of intensities in the matrices tables is not clear: 
Q5: Do the matrices tables (e.g. pg.matrix) contain unnormalized, "normalized", or MaxLFQ-normalized intensities?
Q6: Is it possible to set this (Q4)? I.e. I want to make sure I take MaxLFQ-normalized intensities but like the matrix report due to its simplicity for routine applications. 

The section on matrices outputs states "They are filtered at 1% FDR, using global q-values for protein groups and both global and run-specific q-values for precursors."
Q7: 1% FDR is the default setting in DIA-NN for precursors. When changing that, is the matrix output still filtered at 1% despite the different setting? (and the updated setting is only reflected in the main report?)
Q8: Since this is explicitly stated for the matrix output - what is it for the main report? I presume the protein quantities found in the main report are also based on all precursors that pass the 1% FDR for the same criteria (protein groups, global and run specific q-values for precursors)- correct? In this sense, the protein intensities in the matrix output are identical to one of the three intensities on the main report - correct?

**Comment by vdemichev** [comment, marked as answer]
Q1. Yes.
Q2. (ii)
Q3. Yes.
Q4. Global normalisation is described in DIA-NN paper. Not recommended.
Q5. Normalised, unless normalisation is disabled.
Q7. Yes. Can use --matrix-qvalue to adjust matrix q-value.
Q8. Yes.

**Reply by JM-Bader** [reply]
Thanks for the swift reply.
What about Q6? Is it possible to have the MaxLFQ-normalized intensities in the matrix reports?

**Reply by vdemichev** [reply]
They are by default MaxLFQ and normalised.

**Reply by KR-15** [reply]
Hi @vdemichev @JM-Bader , excellent questions that you asked @JM-Bader and I just want to pick up on what has been said above. 

What I have understand when comparing the intensity values from the main report compared with the pg.matrix report is that the main report show the 'normalized' intensities whereas the pg_matrix shows the maxLFQ-normalized intensities. I hope I am not saying anything wrong but it is what I have understand so far.

I have myself a question, in the paper it is written: 

> A ‘top 3’ method is eventually used for protein quantification: intensities of protein groups are calculated as sums of the intensities of the three most abundant precursors identified with a q-value lower than a given threshold in a particular acquisition.

Is this still the case in new version of DIA-NN? Because I think I have seen somewhere that it only take TOP1 best precursor (after all scoring) for protein quantification but maybe I have misunderstood something.

Also, when it is written the following:

> Top pN precursors are selected, where N is the average number of identifications passing the q-value threshold and p is between
0 and 1. Sums of the intensities of these precursors are calculated and used for normalization (that is, the levels of all precursors are scaled to make these quantities equal in different acquisitions).

What's "p" ?

Thanks a lot ! 

Best,

KR

**Reply by st-pf** [reply]
> They are by default MaxLFQ and normalised.

to add on to the question of KR, the values of the pq.matrix are normalized AND MaxLFQ? (just want to be sure because of the the reply to Q5 above)

Thanks a lot for this great software!

**Reply by vdemichev** [reply]
pg_matrix is normalised & MaxLFQ, unless you disable (can be done separately) either using the respective command line options.

**Reply by st-pf** [reply]
perfect, thank you so much! greetings!
